packages/appengine/src/loadBalancer/details/details.html

```diff
@@ -27,13 +27,14 @@ <h3 class="horizontal middle space-between flex-1" select-on-dbl-click>{{ctrl.lo
             Load Balancer Actions <span class="caret"></span>
           </button>
           <ul class="dropdown-menu" uib-dropdown-menu role="menu">
-            <li class="disabled"><a>Edit Load Balancer</a></li>
-            <li ng-if="ctrl.canDeleteLoadBalancer()">
-              <a href ng-click="ctrl.deleteLoadBalancer()">Delete Load Balancer</a>
+            <li
+              uib-tooltip="Editing a load balancer through Spinnaker is currently not supported."
+              class="disabled"
+            >
+              <a href>Edit Load Balancer</a>
             </li>
             <li
-              ng-if="!ctrl.canDeleteLoadBalancer()"
-              uib-tooltip="You cannot delete a default service."
+              uib-tooltip="Deleting a load balancer through Spinnaker is currently not supported."
```

Comment:
UI tooltip is good idea. Can we change the message from:
`is currently not supported`
to something like:
`is disable due to THD policy`